.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Create a bug report to help us improve
 title: ''
-label: needs triage
+labels: needs triage
```

Comment:
Reviewers: No need to put `"` here as we didn't have it in the past https://github.com/PyTorchLightning/pytorch-lightning/blob/0e6ee9c39dfb1608eb3ffef67f962a725ad935a5/.github/ISSUE_TEMPLATE/bug_report.md?plain=1.